src/load_inputs/load_resources_data.jl

```diff
@@ -444,34 +444,56 @@ function validate_resources(resources::Vector{T}) where T <: AbstractResource
 end
 
 """
-    create_resources(setup::Dict, case_path::AbstractString)
+    create_resources(setup::Dict, resources_path::AbstractString)
 
-Function that loads and scales resources data from folder specified in `setup["ResourcePath"] and returns an array of GenX resources.
+Function that loads and scales resources data from folder specified in resources_path and returns an array of GenX resources.
 
 # Arguments
 - `setup (Dict)`: Dictionary containing GenX settings.
-- `case_path (AbstractString)`: The path to the case.
+- `resources_path (AbstractString)`: The path to the resources folder.
 
 # Returns
 - `resources (Vector{<:AbstractResource})`: An array of scaled resources.
 
 """
-function create_resource_array(setup::Dict, case_path::AbstractString)
+function create_resource_array(setup::Dict, resources_path::AbstractString)
     scale_factor = setup["ParameterScale"] == 1 ? ModelScalingFactor : 1.0
     
-    resources_folder = setup["ResourcePath"]
-    resources_folder = joinpath(case_path,resources_folder)
-    
     # get filename and GenX type for each type of resources available in GenX
     resources_info = _get_resource_info()
 
     # load each resource type, scale data and return array of resources
-    resources = create_resource_array(resources_folder, resources_info, scale_factor)
+    resources = create_resource_array(resources_path, resources_info, scale_factor)
     # validate input before returning resources
     validate_resources(resources)
     return resources
 end
 
+
+"""
+    validate_policy_files(resource_policies_path::AbstractString, setup::Dict)
+
+Validate the policy files by checking if they exist in the specified folder and if the setup flags are consistent with the files found.
+
+# Arguments
+- `resource_policies_path::AbstractString`: The path to the policy files.
+- `setup::Dict`: Dictionary containing GenX settings.
+
+# Returns
+- warning messages if the polcies are set to 1 in settings but the files are not found in the resource_policies_path.
+
+"""
+function validate_policy_files(resource_policies_path::AbstractString, setup::Dict)
+    policyfile_info = _get_policyfile_info()
+    for (filename, setup_param) in values(policyfile_info)
+        if setup[setup_param] == 1 && !isfile(joinpath(resource_policies_path, filename))
```

Comment:
.... I hope that we can change policies from `0`/`1` to `false`/`true` in the future. Or for policies with three or more options (looking at U, UCommit) have them be descriptive strings which translate internally into named constants or enums or something.